tools/UpdatePaths/Scripts/74651_apc_to_apc_helpers.txt

```diff
@@ -0,0 +1,19 @@
+#comment Repaths subtypes and some commonly used properties of apc to apc helpers.
+
+/obj/machinery/power/apc/unlocked : /obj/machinery/power/apc, /obj/effect/mapping_helpers/apc/unlocked{@OLD}
```

Comment:
your `{@OLD}` appears to be on the wrong thing, this will nuke the dir and pixel shift of the APC and give the mapping helper junk variables that it doesn't actually have. put it on the APC so it actually inherits the right thing.

Reply:
Works fine

Reply:
i didn't see you include the solution it's just doing it like `/obj/machinery/power/apc{@OLD;name=@SKIP}